openbb_terminal/common/behavioural_analysis/reddit_view.py

```diff
@@ -323,3 +326,81 @@ def display_due_diligence(
         print_and_record_reddit_post({}, sub)
     if not subs:
         console.print(f"No DD posts found for {ticker}\n")
+
+
+@log_start_end(log=logger)
+@check_api_key(
+    [
+        "API_REDDIT_CLIENT_ID",
+        "API_REDDIT_CLIENT_SECRET",
+        "API_REDDIT_USERNAME",
+        "API_REDDIT_USER_AGENT",
+        "API_REDDIT_PASSWORD",
+    ]
+)
+def display_reddit_sent(
+    ticker: str,
+    sort: str,
+    limit: int,
+    graphic: bool,
+    time_frame: str,
+    full_search: bool,
+    subreddits: str,
+    export: str = "",
+):
+    """Determine Reddit sentiment about a search term
+    Parameters
+    ----------
+    ticker: str
+        The ticker being search for in Reddit
+    sort: str
+        Type of search
+    limit: str
+        Number of posts to get at most
+    graphic: bool
+        Displays box and whisker plot
+    time_frame: str
+        Time frame for search
+    full_search: bool
+        Enable comprehensive search for ticker
+    subreddits: str
+        Comma-separated list of subreddits
+    """
+
+    posts = reddit_model.get_posts_about(ticker, limit, sort, time_frame, subreddits)
+    post_data = []
+    polarity_scores = []
+
+    if not posts:
+        console.print(f"No posts for {ticker} found")
+        return
+
+    console.print("Analyzing each post...")
+    for p in tqdm(posts):
+        texts = [p.title, p.selftext]
+        if full_search:
+            tlcs = reddit_model.get_comments(p)
+            texts.extend(tlcs)
+        preprocessed_text = reddit_model.clean_reddit_text(texts)
+        sentiment = reddit_model.get_sentiment(preprocessed_text)
+        polarity_scores.append(sentiment)
+        post_data.append([p.title, sentiment])
+
+    avg_polarity = sum(polarity_scores) / len(polarity_scores)
+
+    columns = ["Title", "Polarity Score"]
+    df = pd.DataFrame(post_data, columns=columns)
+
+    if graphic:
```

Comment:
[critical] this is the only critical comment for the PR.
the way the plot is created and used here leads to axes overlapping each other on the same figure if you call the command multiple times consecutively. 
<img width="626" alt="image" src="https://user-images.githubusercontent.com/11668535/165490867-850040d2-418e-434b-981a-f0a2c7aaf8ab.png">

in addition there's no support for external axes which makes it impossible to use the chart in notebooks and reports.

to resolve this you'll need to create a use seaborn itself in external axis mode. there are a couple of examples in the code of how the chart that uses seaborn can be implemented. imo this is the simplest one:
https://github.com/OpenBB-finance/OpenBBTerminal/blob/71cd1023d4d41afff50d89c6b7869027837564b9/openbb_terminal/stocks/comparison_analysis/yahoo_finance_view.py#L203-L228

note the `ax=ax` kwarg going to the sns call and use of the `theme` helper

Reply:
Great catch here! Hopefully this is fixed by [ac8d2c6](https://github.com/OpenBB-finance/OpenBBTerminal/pull/1744/commits/ac8d2c60aaf03f4b63a1082116bf21756b172dd9), but let me know if I did anything wrong.